src/modules/ipfix/IpfixPrinter.cpp

```diff
@@ -67,15 +67,17 @@ void PrintHelpers::printPort(InformationElement::IeInfo type, IpfixRecord::Data*
 		return;
 	}
 	if (type.length == 2) {
-		int port = ((uint16_t)data[0] << 8)+data[1];
+		// get the first 2 bytes from data and convert them from network endianess to host endianess
+		int port = ntohs( (uint16_t)*( (uint16_t *)(&data[0]) ) );
```

Comment:
* Is the cast to `uint16_t` after derefencing necessary?
* Why are you dereferencing (using `[0]`) and then referencing (using `&`) the data variable?

Reply:
The cast to uint16_t can indeed be removed. I use "&data[0]" instead of "data" because I think the code looks clearer this way: it is more clear that I want to use the first 2 bytes in data and it is consistent with &data[i+0] and &data[i+2]